devnet-sdk/shell/env/devnet.go

```diff
@@ -91,6 +116,13 @@ func (d *DevnetEnv) GetChain(chainName string) (*ChainConfig, error) {
 	}, nil
 }
 
+func (d *DevnetEnv) Control() (surface.ControlSurface, error) {
+	if d.ctrl == nil {
+		return nil, fmt.Errorf("devnet is not controllable")
+	}
+	return d.ctrl()
```

Comment:
Why `d.ctrl` and `d.ctrl()`?

Reply:
d.ctrl is a function value, d.ctrl() is how we get the actual ControlSurface